### ISSUE TITLE
Versioning of CentOS warning and content

### ISSUE THREAD

**Issue by guardrex** [dotnet/AspNetCore.Docs#32308]
### Description

@Rick-Anderson @wadepickett @tdykstra 

Per https://github.com/dotnet/AspNetCore.Docs/pull/32307 and at least here and over in ...

https://learn.microsoft.com/aspnet/core/host-and-deploy/linux-apache

... version the CentOS warning so that it doesn't appear here ***forever***. I suppose it can be versioned out (or removed entirely) for >=9.0 with a short section at the ends of these articles appearing for >=9.0 with ...

* Remark CentOS is EOL.
* Cross-link to the EOL CentOS guidance using the link that's in the CAUTION.
* Instruction to load a version of the article <9.0 for prior CentOS guidance.

I'll also check the entire repo for CentOS guidance and act on it.

I'll ping for review on the PR when it goes up.

Does that all sound ok?

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-8.0&branch=pr-en-us-32307

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/host-and-deploy/webassembly.md

### Document ID

250e7458-3c1d-0117-261d-c03ac136f16a

### Article author

@guardrex

**Comment by tdykstra** [comment]
>Does that all sound ok?

Sounds OK to me.

**Comment by guardrex** [comment]
I have the Blazor side of this covered for now until I chat with Dan and Artak about it. They'll need to coordinate a different OS for our Apache-hosted standalone WASM coverage.

In the process, I fixed a couple of spots where docs used CentOS as the example of a case sensitive file system. I changed those to Ubuntu.

Otherwise, I'm removing my assignment here because I can't address the linux-apache rewrite and CentOS mentions using the INCLUDE versioning approach.

**Comment by guardrex** [comment]
Reassigning myself just in case this will receive special attention in Blazor docs.

@danroth27 & @mkArtakMSFT ... 

### TL;DR ... The problem is that CentOS is EOL, but we have this content right now ...

https://learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-8.0#apache

Should we work on a replacement for that coverage (Ubuntu?), or should we cut it 🔪 without replacement? I think the former (Ubuntu) makes more sense.
